test/libsolidity/SemanticTest.cpp

```diff
@@ -122,6 +124,14 @@ SemanticTest::SemanticTest(
 	}
 }
 
+void SemanticTest::initializeBuiltins()
+{
+	m_builtins["smoke_test"] = [](FunctionCall const&) -> std::optional<bytes>
```

Comment:
Don't we typically use camel case for builtins?

Also, make sure there are no name clashes (unlikely but better safe than sorry :)).
```suggestion
	solAssert(m_builtins.count("smokeTest") == 0, "");
	m_builtins["smokeTest"] = [](FunctionCall const&) -> std::optional<bytes>
```

Reply:
This is the first builtin to be added, but the assert is good to have, yes.

Reply:
Ah, good point. I thought that the existing "directives" were builtins but I see that it's a new thing.